packages/components/src/mobile/badge/README.md

```diff
@@ -0,0 +1,31 @@
+# Badge
+
+The Badge component is designed to be wrapped around another component. It add a "badge" with some text to the child component's upper left.
```

Comment:
> add

Typo perhaps? Replace with "adds"?

Reply:
A typo indeed, thank you for spotting that! I've updated.